api/src/pcapi/connectors/api_adresse.py

```diff
@@ -169,19 +183,31 @@ def get_municipality_centroid(
             id="06029",
             label="Cannes",
             postcode="06400",
+            citycode="06029",
             score=0.9549627272727272,
             latitude=43.555468,
             longitude=7.004585,
+            city="Cannes",
+            street="Cannes",
```

Comment:
Question : est-ce bien le comportement de l'API Adresse : nom de la ville dans la rue lorsqu'on demande le centroïde ? Ne serait-il pas pertinent de laisser ce champ vide dans la base de données ?

Reply:
Oui c'est bien ce que retourne l'API, pour le centroïde:

```
      "properties": {
        "label": "Cannes",
        "score": 0.9549627272727272,
        "id": "06029",
        "type": "municipality",
        "name": "Cannes",
        "postcode": "06400",
        "citycode": "06029",
        "x": 1023559.08,
        "y": 6281139.85,
        "population": 72435,
        "city": "Cannes",
        "context": "06, Alpes-Maritimes, Provence-Alpes-Côte d'Azur",
        "importance": 0.50459,
        "municipality": "Cannes"
      }
```

Pour une demande précise:
```
      "properties": {
        "label": "Avenue de Grasse 06400 Cannes",
        "score": 0.7079445454545453,
        "id": "06029_1560",
        "name": "Avenue de Grasse",
        "postcode": "06400",
        "citycode": "06029",
        "x": 1023577.18,
        "y": 6281720.9,
        "city": "Cannes",
        "context": "06, Alpes-Maritimes, Provence-Alpes-Côte d'Azur",
        "type": "street",
        "importance": 0.78739,
        "street": "Avenue de Grasse"
      }
```

In fine ces données seront affichées à l'utilisateur, je ne suis pas sur que ne rien afficher soit mieux qu'afficher le nom de la ville

Reply:
D'autre part, il faut savoir que l'utilisateur n'aura pas de formulaire complet côté front, seulement un champ autocomplété via un premier appel de l'api adresse dans le front, donc c'est de toute manière ce qu'on recevra du front

Reply:
Je reste sur ma position, la ville n'est pas l'adresse :slightly_smiling_face: 

D'une part tu montres bien pour une demande précise : 
```
        "type": "street",
        "name": "Avenue de Grasse",
        "street": "Avenue de Grasse"
```
Il s'agit bien là du numéro et de la rue dans `name` car on a un type `street`, mais on pourrait aussi lire les champs `housenumber` (si type `housenumber`), `street`, `locality`. Ça fonctionne jusqu'ici donc gardons `name`.

D'autre part pour le centroïde, on a pour ces mêmes clés : 
```
        "type": "municipality",
        "name": "Cannes",
```
Ici, `name` correspond donc à la commune, comme indiqué dans `type` : je ne vois pas de raison de remplir le champ `street` dans notre base de données avec une information que l'API ne nous a pas retournée comme étant la rue. Et côté front ça ne devrait pas non plus considérer ce champ comme la rue, si tel est le cas.

Tu parles de « ne rien afficher », mais on aura bien le code postal et la ville, juste pas de rue. Je ne pense pas qu'afficher « Cannes, 06400 Cannes » soit mieux que « 06400 Cannes ».

Reply:
Discuté ensemble, on rend nullable street pour les rares cas où on fallback sur le centroïde de la commune. J'ai mis à jour avec un test supplémentaire